chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/AddedToRoom.kt

```diff
@@ -0,0 +1,33 @@
+package com.pusher.chatkit.rooms.state
+
+import com.pusher.chatkit.state.ChatkitState
+import org.reduxkotlin.reducerForActionType
+
+internal data class AddedToRoom(
```

Comment:
let's use consistent naming with [the public api](https://github.com/pusher/chatkit-android/pull/174/files#diff-7b362c0cc548b3a5a696fdd71ca57fbbR25), unless deviations are justified

Reply:
👍  Have updated AddedToRoom -> JoinedRoom and RemovedFromRoom -> LeftRoom